src/nikhilMJun20/StringFrequency.java

```diff
@@ -0,0 +1,49 @@
+/*
+Assignment-37 : 2nd Aug'2020
+
+Input : Hi Globant techno Hi Hello Globant techno techno Globant techno
+FR - 1 : freq of each word
+FR - 2 : Maxmimum repeated word and its frequency
+
+Output : Hi -> 2
+Globant -> 3
+techno -> 4
+Hello -> 1
+Maximum repeated word is techno having frequency 4.
+*/
+
+package nikhilMJun20;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.LinkedHashMap;
+
+public class StringFrequency {
+	void findFrequency(String input) {
+		ArrayList<String> strList = new ArrayList<String> (Arrays.asList(input.split(" ")));
```

Comment:
--logic is correct
--However here you can directly use array as well String[] array = str.split(" ");
--Nothing wrong in using ArrayList as well.